nrcemt/qeels/gui/main_window.py

```diff
@@ -213,6 +214,16 @@ def canvas_click(self, x, y):
         selected_plasmon = self.plasmon_array[self.radio_variable.get()]
         selected_plasmon.x.set(x)
         selected_plasmon.y.set(y)
+        row = int(self.radio_variable.get()/2)
+
+        x1 = self.plasmon_array[row*2].x.get()
+        y1 = self.plasmon_array[row*2].y.get()
+        x2 = self.plasmon_array[row*2+1].x.get()
+        y2 = self.plasmon_array[row*2+1].y.get()
+
+        if np.all([x1, y1, x2, y2]) > 0:
```

Comment:
I don't think it this works how you think it does. np.all returns boolean value representing whether all values were non-zero. The `> 0` here is not actually comparing numerically, it is just returning `True` because `True > 0` evaluates to `True`.

Reply:
should probs just be `np.all` without `> 0`

Reply:
also, I think you just use the stock python `all` function for this `all([x1, y1, x2, y2])`, no need for numpy